test/queue_test.js

```diff
@@ -104,7 +104,8 @@ suite("Queue", function() {
     // Don't actually want to change the meta-data as it would affect the
     // createQueue test case...
     return queue.setMetadata(queueName, {
-      purpose:  'testing'
+      purpose:  'testing',
+      applicationName: 'fast-azure-storage',
```

Comment:
Ah, okay... Can we include the spaces here too then? We want to include them because that's part of the test case :)